csharp_sdk_fix/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+# Note this is a separate file to include as early as possible in the root project when it is needed
+
+# This can go away when this cmake issue is resolved and we move to a version of cmake with that fix https://gitlab.kitware.com/cmake/cmake/-/issues/23513
```

Comment:
>https://gitlab.kitware.com/cmake/cmake/-/issues/23513 [](http://example.com/codeflow?start=101&length=53)

I see that as of 3 weeks ago there was still no significant traction. A shame.

Reply:
Yeah there is kind of a post on how it could be fixed...if I had nothing else to do I'd consider taking a stab at it haha because this hack is less-than-ideal.